controllers/storagecluster/external_resources.go

```diff
@@ -326,6 +344,82 @@ func (obj *ocsExternalResources) ensureDeleted(r *StorageClusterReconciler, inst
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) createStorageClassClaim(instance *ocsv1.StorageCluster) error {
```

Comment:
rename function to `createDefaultStorageClassClaims`

Reply:
done